dependencies.gradle

```diff
@@ -21,6 +21,9 @@ ext {
     //Amplitude
     amplitudeVersion = '2.23.2'
 
+    // Clever Tap
```

Comment:
Without space please

Reply:
Nooooooo!

Reply:
![image](https://user-images.githubusercontent.com/18698430/72883655-c95acc80-3d15-11ea-8174-fef898128b82.png)
Some have it and other ones no.